lib/plugins/pre_commit/checks/jshint.rb

```diff
@@ -23,7 +23,7 @@ def self.description
       private
 
       def context
-        @context ||= ExecJS.compile(File.read(linter_src) << <<-JAVASCRIPT)
+        @context ||= ExecJS.compile("global = this;" << File.read(linter_src) << <<-JAVASCRIPT)
```

Comment:
This is weird. Without this, lodash's `runInContext` method dies because it's looking at an empty object that does not have Array, Object, etc defined. There is a lot of "root variable" detection code presumably coming from webpack (or whatever javascript transpiler jshint uses).

This `global = this` could alternatively become `window = this`. As best I understand it, the problem is that TheRubyRacer (the js engine I was testing with) does not define `global` *or* `window`, one of which needs to be defined for jshint to work.